packages/shopping/package.json

```diff
@@ -81,7 +82,7 @@
     "@types/mocha": "7.0.2",
     "@types/node": "12.12.47",
     "concurrently": "5.2.0",
-    "loopback4-example-recommender": "1.1.1",
+    "loopback4-example-recommender": "file:../recommender",
```

Comment:
IIRC, `lerna` allows both style.

Reply:
Doesn't seem to be working anymore. Take a look here - https://github.com/strongloop/loopback4-example-shopping/pull/760.